src/executor/graph_executor.cc

```diff
@@ -2061,25 +2061,26 @@ Executor *Executor::Bind(nnvm::Symbol symbol,
                          const std::vector<OpReqType> &grad_req_type,
                          const std::vector<NDArray> &aux_states,
                          Executor* shared_exec) {
-  auto exec = new exec::GraphExecutor(symbol);
+  // Verbosity level: If =1 logs the name of subgaph backend
+  //                  If >1 then additional details regarding subgraph backend is logged
   static int verbose = dmlc::GetEnv("MXNET_SUBGRAPH_VERBOSE", 1);
   std::vector<NDArray> tmp_in_args = in_args;
   std::vector<NDArray> tmp_arg_grad_store = arg_grad_store;
   std::vector<OpReqType> tmp_grad_req_type = grad_req_type;
   std::vector<NDArray> tmp_aux_states = aux_states;
 
-  if (!exec->subgraph_property().empty()) {
-    const auto& backend_name = exec->subgraph_property();
+  const auto& backend_name = dmlc::GetEnv("MXNET_SUBGRAPH_BACKEND", exec::GetDefaultSubgraphBackend());
+  if (!backend_name.empty()) {
```

Comment:
we should also include the check `backend_name ==  "None"` or we can do something like this before the condition https://github.com/apache/incubator-mxnet/blob/3b0b9dc9d865ee0b1456e8f400f6f4a25c77b5db/src/executor/graph_executor.cc#L59-L60

Reply:
I prefer we dont duplicate code/functionality if at all possible.  Since we create `exec` with `new`, and then use it to get the `backend_name`, why dont we just call delete on `exec` before line 2080 when creating a new executor from the partitioned symbol?

Reply:
yeah I agree, we can just use `delete exec` for both functions